src/main/java/org/sonar/plugins/stash/StashPluginUtils.java

```diff
@@ -1,20 +1,20 @@
 package org.sonar.plugins.stash;
 
 import org.sonar.api.issue.Issue;
+import org.sonar.api.resources.Project;
 
 import java.math.RoundingMode;
 import java.text.DecimalFormat;
 import java.text.DecimalFormatSymbols;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Map'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)